redux/actions/forgotPassword.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+import * as actions from "./types";
+
+const posting = axios.create({baseURL: `${process.env.REACT_APP_BACKEND_URL}`});
```

Comment:
As all need this functionality, can't we make it global to prevent repeating ourselves